test/parallel/sdk.API.submitableExtrinsic.test.ts

```diff
@@ -171,7 +171,7 @@ test("check claimRewards", async () => {
   expect(userTokenAfterClaiming.rewardsAlreadyClaimed).bnGt(BN_ZERO);
 });
 
-test("check mintLiquidity", async () => {
+test.skip("check mintLiquidity", async () => {
```

Comment:
basically, SDK is not avl for that funcs anymore, right?

Reply:
Yes, I think so